exts/virtio_blk/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "virtio_blk"
```

Comment:
Can we please avoid unnecessary abbreviations? `virtio_block` and `VirtIOBlock` looks much better and understandable for the human eye. These aren't competition in who's faster programmer.

Yes, I know this is name used by Qemu - doesn't mean we have to copy every bad habit.

Reply:
I agree that it is pretty ugly. I was just adhering to conformance with the Virtio spec and Qemu. Don't mind making it more legible.

Reply:
commit 921d8f2